apps/web-dashboard/src/components/MagicBento/MagicBento.css

```
@@ -0,0 +1,298 @@
+:root {
+  --hue: 27;
+  --sat: 69%;
+  --white: hsl(0, 0%, 100%);
+  --purple-primary: rgba(0, 245, 212, 1);
+  --purple-glow: rgba(0, 245, 212, 0.2);
+  --purple-border: rgba(0, 245, 212, 0.8);
+  --border-color: #2F293A;
+  --background-dark: #120F17;
+  color-scheme: light dark;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)